services/ui-src/src/components/overlays/EntityDetailsMultiformOverlay.tsx

```diff
@@ -322,10 +336,11 @@ export const EntityDetailsMultiformOverlay = ({
         typeof header === "object" ? header.hiddenName : header;
       const isEnabled = formEnableDetails[formId];
       const isComplete = formHasComplianceDetails[formId];
+      const is438206Form = formId === "planCompliance438206";
 
       switch (headerName) {
         case "Status": {
-          if (isEnabled) {
+          if (is438206Form || !isComplete) {
```

Comment:
per design: 
- display icon for 438.206 in all instances
- display icon for 438.68 only if incomplete